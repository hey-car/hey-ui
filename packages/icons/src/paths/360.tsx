/* eslint-disable unicorn/filename-case */
/* eslint-disable @typescript-eslint/naming-convention */

import React from 'react';

import createSvgIcon from '../utils/createSvgIcon';

export const _360 = createSvgIcon(
  <path
    d="M6.924 8.018C7.168 8.07 7.432 8.096 7.716 8.096C8.008 8.096 8.272 8.06 8.508 7.988C8.744 7.912 8.942 7.806 9.102 7.67C9.266 7.534 9.39 7.374 9.474 7.19C9.558 7.006 9.6 6.806 9.6 6.59C9.6 6.246 9.506 5.964 9.318 5.744C9.134 5.52 8.872 5.376 8.532 5.312L9.438 4.442V3.536L6.42 3.536V4.598H7.86L6.918 5.51L7.224 6.212C7.324 6.18 7.46 6.164 7.632 6.164C7.82 6.164 7.964 6.2 8.064 6.272C8.168 6.344 8.22 6.446 8.22 6.578C8.22 6.714 8.164 6.82 8.052 6.896C7.944 6.972 7.79 7.01 7.59 7.01C7.398 7.01 7.2 6.986 6.996 6.938C6.792 6.886 6.6 6.814 6.42 6.722L6.306 7.79C6.474 7.89 6.68 7.966 6.924 8.018ZM10.7335 7.646C11.0575 7.946 11.5075 8.096 12.0835 8.096C12.4275 8.096 12.7295 8.028 12.9895 7.892C13.2535 7.756 13.4555 7.572 13.5955 7.34C13.7395 7.104 13.8115 6.838 13.8115 6.542C13.8115 6.262 13.7475 6.014 13.6195 5.798C13.4955 5.582 13.3275 5.416 13.1155 5.3C12.9035 5.18 12.6655 5.12 12.4015 5.12C12.1975 5.12 12.0255 5.162 11.8855 5.246C12.0415 4.99 12.2495 4.724 12.5095 4.448C12.7695 4.172 13.0955 3.876 13.4875 3.56L11.8795 3.512C11.3675 3.924 10.9675 4.384 10.6795 4.892C10.3915 5.4 10.2475 5.9 10.2475 6.392C10.2475 6.928 10.4095 7.346 10.7335 7.646ZM12.3715 6.884C12.2875 6.968 12.1795 7.01 12.0475 7.01C11.9195 7.01 11.8115 6.968 11.7235 6.884C11.6395 6.796 11.5975 6.688 11.5975 6.56C11.5975 6.428 11.6395 6.32 11.7235 6.236C11.8115 6.152 11.9195 6.11 12.0475 6.11C12.1795 6.11 12.2875 6.152 12.3715 6.236C12.4555 6.32 12.4975 6.428 12.4975 6.56C12.4975 6.688 12.4555 6.796 12.3715 6.884ZM16.3311 6.626C16.2711 6.81 16.1771 6.902 16.0491 6.902C15.9211 6.902 15.8251 6.81 15.7611 6.626C15.7011 6.442 15.6711 6.156 15.6711 5.768C15.6711 5.38 15.7011 5.094 15.7611 4.91C15.8251 4.726 15.9211 4.634 16.0491 4.634C16.1771 4.634 16.2711 4.726 16.3311 4.91C16.3951 5.094 16.4271 5.38 16.4271 5.768C16.4271 6.156 16.3951 6.442 16.3311 6.626ZM14.7051 7.484C15.0251 7.892 15.4731 8.096 16.0491 8.096C16.6251 8.096 17.0711 7.892 17.3871 7.484C17.7071 7.076 17.8671 6.504 17.8671 5.768C17.8671 5.032 17.7071 4.46 17.3871 4.052C17.0711 3.644 16.6251 3.44 16.0491 3.44C15.4731 3.44 15.0251 3.644 14.7051 4.052C14.3891 4.46 14.2311 5.032 14.2311 5.768C14.2311 6.504 14.3891 7.076 14.7051 7.484ZM20 13.9899C20 14.2 19.7766 14.9311 18.1781 15.7287C17.108 16.2627 15.6689 16.6793 14 16.8686V18.88C18.5645 18.4176 22 16.404 22 13.9899C22 11.2341 17.5228 9 12 9C6.47715 9 2 11.2341 2 13.9899C2 16.1963 4.86995 18.0683 8.84873 18.727L7.95619 19.6195L9.3704 21.0337L12.0337 18.3704L12.7408 17.6633L12.0337 16.9562L9.3704 14.2929L7.95619 15.7071L8.96694 16.7179C7.7352 16.4966 6.66423 16.149 5.82191 15.7287C4.22337 14.9311 4 14.2 4 13.9899C4 13.7798 4.22337 13.0487 5.82191 12.2511C7.29967 11.5137 9.48118 11 12 11C14.5188 11 16.7003 11.5137 18.1781 12.2511C19.7766 13.0487 20 13.7798 20 13.9899Z"
    fill-rule="evenodd"
  />,
  '360',
);
