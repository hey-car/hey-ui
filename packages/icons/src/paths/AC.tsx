/* eslint-disable unicorn/filename-case */
/* eslint-disable @typescript-eslint/naming-convention */
import React from 'react';

import createSvgIcon from '../utils/createSvgIcon';

export const AC = createSvgIcon(
  <path d="M18.7868 6.92869L20.1597 6.13605L21.1597 7.8681L19.7865 8.66093L21.5529 9.13422L21.0352 11.0661L17.337 10.0751L13.9994 12.0021L17.337 13.929L21.0352 12.9381L21.5529 14.8699L19.7865 15.3432L21.1597 16.1361L20.1597 17.8681L18.7868 17.0755L19.26 18.8413L17.3281 19.359L16.3373 15.6612L13.0005 13.7347V17.5879L15.7076 20.295L14.2934 21.7092L13.0005 20.4163V22.0021H11.0005V20.4163L9.70761 21.7092L8.2934 20.295L11.0005 17.5879V13.7335L7.66296 15.6604L6.67194 19.359L4.74009 18.8413L5.21347 17.0747L3.83918 17.8681L2.83918 16.1361L4.21277 15.343L2.44719 14.8699L2.96483 12.9381L6.66225 13.9288L9.99944 12.0021L6.66227 10.0754L2.96483 11.0661L2.4472 9.13423L4.21277 8.66115L2.83918 7.8681L3.83918 6.13605L5.21346 6.92949L4.74009 5.16282L6.67194 4.64518L7.66295 8.3437L11.0005 10.2706V6.41628L8.2934 3.70918L9.70761 2.29497L11.0005 3.58786V2.00208H13.0005V3.58787L14.2934 2.29497L15.7076 3.70918L13.0005 6.4163V10.2694L16.3373 8.3429L17.3281 4.64518L19.26 5.16282L18.7868 6.92869Z" />,
  'AC',
);
