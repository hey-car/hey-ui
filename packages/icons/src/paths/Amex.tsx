import React from 'react';

import createSvgIcon from '../utils/createSvgIcon';

export const Amex = createSvgIcon(
  <>
    <path
      d="M.25.25h23.5v23.5H.25z"
      fill="transparent"
      stroke="#EAEAEA"
      strokeWidth=".5"
    />
    <path
      clipRule="evenodd"
      d="M4.257 9.692 2 14.302h2.702l.335-.735h.766l.335.735h2.974v-.561l.265.56h1.539l.265-.572v.573h6.185l.753-.716.704.716 3.177.006-2.264-2.298L22 9.692h-3.128l-.732.703-.682-.703h-6.73l-.577 1.19-.592-1.19H6.863v.542l-.3-.542H4.257Zm.523.655h1.317l1.497 3.126v-3.126h1.443l1.156 2.242 1.066-2.242h1.436v3.308h-.874l-.007-2.592-1.273 2.592h-.782l-1.28-2.592v2.592H6.68l-.34-.742H4.5l-.34.741h-.963l1.583-3.307Zm12.266 0h-3.552v3.306h3.497l1.128-1.096 1.086 1.096h1.136L18.69 12.01l1.65-1.663h-1.086l-1.121 1.083-1.087-1.083Zm-11.625.56-.607 1.32h1.213l-.606-1.32Zm8.95.73v-.604h2.217l.967.965-1.01.971h-2.174v-.66h1.938v-.672h-1.938Z"
      fill="#044BBC"
      fillRule="evenodd"
    />
  </>,
  'Amex',
);
