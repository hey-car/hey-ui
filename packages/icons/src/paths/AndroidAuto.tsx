import React from 'react';

import createSvgIcon from '../utils/createSvgIcon';

export const AndroidAuto = createSvgIcon(
  <path
    d="M10.8338 4.94365L12 3L13.1662 4.94365L19.8 16L21 18H18.6676H16.2857L18 21L12 18.25L6 21L7.71428 18H5.33238L3 18L4.2 16L10.8338 4.94365ZM15.1429 16L12 10.5L8.85714 16H6.53238L12 6.8873L17.4676 16H15.1429Z"
    fillRule="evenodd"
  />,
  'AndroidAuto',
);
