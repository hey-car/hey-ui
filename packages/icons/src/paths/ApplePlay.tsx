import React from 'react';

import createSvgIcon from '../utils/createSvgIcon';

export const ApplePlay = createSvgIcon(
  <path d="M4.75775 12C4.75775 7.58172 8.33948 4 12.7578 4C15.5305 4 17.9744 5.40977 19.4112 7.55626C19.7184 8.01521 20.3395 8.13822 20.7985 7.83101C21.2574 7.52379 21.3804 6.90269 21.0732 6.44374C19.281 3.76629 16.2258 2 12.7578 2C7.23491 2 2.75775 6.47715 2.75775 12C2.75775 17.5228 7.23491 22 12.7578 22C16.2258 22 19.281 20.2337 21.0732 17.5563C21.3804 17.0973 21.2574 16.4762 20.7985 16.169C20.3395 15.8618 19.7184 15.9848 19.4112 16.4437C17.9744 18.5902 15.5305 20 12.7578 20C8.33948 20 4.75775 16.4183 4.75775 12ZM17.7579 12L10.7579 16V8L17.7579 12Z" />,
  'ApplePlay',
);
