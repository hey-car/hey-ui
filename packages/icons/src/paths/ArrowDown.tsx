import React from 'react';

import createSvgIcon from '../utils/createSvgIcon';

export const ArrowDown = createSvgIcon(
  <path
    clipRule="evenodd"
    d="M13.0004 17.1712L12.9995 3.00049L10.9995 3.00061L11.0004 17.1729L4.22134 10.3939L2.80713 11.8081L11.9995 21.0005L21.1929 11.8071L19.7787 10.3929L13.0004 17.1712Z"
    fillRule="evenodd"
  />,
  'ArrowDown',
);
