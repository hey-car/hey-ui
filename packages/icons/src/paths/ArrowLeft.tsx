import React from 'react';

import createSvgIcon from '../utils/createSvgIcon';

export const ArrowLeft = createSvgIcon(
  <path
    clipRule="evenodd"
    d="M6.82929 13.0009L21 13L20.9999 11L6.82757 11.0009L13.6066 4.22183L12.1924 2.80762L3 12L12.1934 21.1934L13.6076 19.7791L6.82929 13.0009Z"
    fillRule="evenodd"
  />,
  'ArrowLeft',
);
