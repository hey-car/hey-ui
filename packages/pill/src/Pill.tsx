import React from 'react';

import { PillProps } from './Pill.types';

import styles from './styles/default.module.css';

const Pill = React.forwardRef<HTMLElement, PillProps>(
  (
    { children, leftIcon, dataTestId, Component = 'div', ...restProps },
    ref,
  ) => {
    return (
      <Component
        className={styles.pill}
        data-test-id={dataTestId}
        ref={ref}
        {...restProps}
      >
        {leftIcon && <span className={styles.leftIcon}>{leftIcon}</span>}
        {children}
      </Component>
    );
  },
);

export default Pill;
