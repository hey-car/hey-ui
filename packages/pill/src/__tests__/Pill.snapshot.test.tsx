import React from 'react';
import { render } from '@testing-library/react';

import { ABSport as Icon } from '../../../icons/src/paths/ABSport';
import Pill from '../Pill';

const defaultChild = '$100 off';
const dataTestId = 'testId';
const testComponent = 'span';

describe('Pill Snapshots tests', () => {
  it('should match snapshot', () => {
    expect(render(<Pill>{defaultChild}</Pill>)).toMatchSnapshot();
  });

  it('should match data-test-id', () => {
    expect(
      render(<Pill dataTestId={dataTestId}>{defaultChild}</Pill>),
    ).toMatchSnapshot();
  });

  it('should match passed Component', () => {
    expect(
      render(<Pill Component={testComponent}>{defaultChild}</Pill>),
    ).toMatchSnapshot();
  });

  it('should match leftIcon', () => {
    expect(
      render(<Pill leftIcon={<Icon />}>{defaultChild}</Pill>),
    ).toMatchSnapshot();
  });
});
