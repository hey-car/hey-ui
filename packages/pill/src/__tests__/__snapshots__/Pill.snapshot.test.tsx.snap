// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pill Snapshots tests should match data-test-id 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="pill"
        data-test-id="testId"
      >
        $100 off
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="pill"
      data-test-id="testId"
    >
      $100 off
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Pill Snapshots tests should match leftIcon 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="pill"
      >
        <span
          class="leftIcon"
        >
          <svg
            class="svgIcon inherit"
            data-test-id="ABSportIcon"
            focusable="false"
            style="font-size: 24px;"
            viewBox="0 0 24 24"
          >
            <path
              d="M7.20058 5.59889C6.77839 5.91601 6.38849 6.27397 6.03682 6.6669C4.76945 8.08291 4 9.9504 4 12C4 14.0496 4.76945 15.9171 6.03682 17.3331C6.38849 17.726 6.77839 18.084 7.20058 18.4011L5.99942 20.0002C5.47221 19.6042 4.98551 19.1573 4.54655 18.6669C2.96367 16.8984 2 14.5605 2 12C2 9.43944 2.96367 7.10159 4.54655 5.33307C4.9855 4.84262 5.4722 4.39576 5.99942 3.99976L7.20058 5.59889ZM17.9632 6.6669C17.6115 6.27397 17.2216 5.91601 16.7994 5.59889L18.0006 3.99976C18.5278 4.39576 19.0145 4.84262 19.4535 5.33307C21.0363 7.10159 22 9.43944 22 12C22 14.5605 21.0363 16.8984 19.4535 18.6669C19.0145 19.1573 18.5278 19.6042 18.0006 20.0002L16.7994 18.4011C17.2216 18.084 17.6115 17.726 17.9632 17.3331C19.2306 15.9171 20 14.0496 20 12C20 9.9504 19.2306 8.08291 17.9632 6.6669ZM12 17C14.7614 17 17 14.7614 17 12C17 9.23855 14.7614 6.99998 12 6.99998C9.23858 6.99998 7 9.23855 7 12C7 14.7614 9.23858 17 12 17ZM12 19C15.866 19 19 15.866 19 12C19 8.13398 15.866 4.99998 12 4.99998C8.13401 4.99998 5 8.13398 5 12C5 15.866 8.13401 19 12 19ZM9.9705 10.84H9.0955L8.07 13.5H8.9065L9.0745 12.9505H9.9355L10.107 13.5H10.9995L9.9705 10.84ZM9.505 11.547L9.743 12.3205L9.267 12.3205L9.505 11.547ZM12.3923 10.84H11.3143V13.5H12.4973C13.1717 13.5 13.5088 13.2398 13.5088 12.7195C13.5088 12.5631 13.4703 12.429 13.3933 12.317C13.3163 12.2026 13.2102 12.1245 13.0748 12.0825C13.1705 12.0288 13.2428 11.9541 13.2918 11.8585C13.3432 11.7628 13.3688 11.6531 13.3688 11.5295C13.3688 11.3031 13.286 11.1316 13.1203 11.015C12.957 10.8983 12.7143 10.84 12.3923 10.84ZM12.4308 12.933H12.0668V12.443H12.4308C12.6152 12.443 12.7073 12.5246 12.7073 12.688C12.7073 12.8513 12.6152 12.933 12.4308 12.933ZM12.3328 11.897H12.0668V11.407H12.3328C12.5172 11.407 12.6093 11.4886 12.6093 11.652C12.6093 11.8153 12.5172 11.897 12.3328 11.897ZM14.1753 13.4895C14.3573 13.5338 14.5428 13.556 14.7318 13.556C15.0865 13.556 15.3606 13.4801 15.5543 13.3285C15.7503 13.1745 15.8483 12.9586 15.8483 12.681C15.8483 12.4616 15.7771 12.2843 15.6348 12.149C15.4948 12.0136 15.2673 11.9051 14.9523 11.8235C14.824 11.7908 14.7365 11.7605 14.6898 11.7325C14.6431 11.7021 14.6198 11.6625 14.6198 11.6135C14.6198 11.5575 14.6431 11.5166 14.6898 11.491C14.7365 11.463 14.8088 11.449 14.9068 11.449C15.0258 11.449 15.1518 11.4676 15.2848 11.505C15.4178 11.54 15.5298 11.5866 15.6208 11.645L15.7258 10.9975C15.6255 10.9321 15.4971 10.8808 15.3408 10.8435C15.1845 10.8038 15.0258 10.784 14.8648 10.784C14.5311 10.784 14.2698 10.8598 14.0808 11.0115C13.8941 11.1631 13.8008 11.372 13.8008 11.638C13.8008 11.8596 13.8661 12.0381 13.9968 12.1735C14.1275 12.3088 14.334 12.4126 14.6163 12.485C14.7213 12.513 14.8006 12.5375 14.8543 12.5585C14.908 12.5795 14.9465 12.6028 14.9698 12.6285C14.9955 12.6518 15.0083 12.6821 15.0083 12.7195C15.0083 12.8338 14.9021 12.891 14.6898 12.891C14.5405 12.891 14.3865 12.87 14.2278 12.828C14.0715 12.7836 13.9373 12.7253 13.8253 12.653L13.7553 13.3215C13.8533 13.3891 13.9933 13.4451 14.1753 13.4895Z"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        $100 off
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="pill"
    >
      <span
        class="leftIcon"
      >
        <svg
          class="svgIcon inherit"
          data-test-id="ABSportIcon"
          focusable="false"
          style="font-size: 24px;"
          viewBox="0 0 24 24"
        >
          <path
            d="M7.20058 5.59889C6.77839 5.91601 6.38849 6.27397 6.03682 6.6669C4.76945 8.08291 4 9.9504 4 12C4 14.0496 4.76945 15.9171 6.03682 17.3331C6.38849 17.726 6.77839 18.084 7.20058 18.4011L5.99942 20.0002C5.47221 19.6042 4.98551 19.1573 4.54655 18.6669C2.96367 16.8984 2 14.5605 2 12C2 9.43944 2.96367 7.10159 4.54655 5.33307C4.9855 4.84262 5.4722 4.39576 5.99942 3.99976L7.20058 5.59889ZM17.9632 6.6669C17.6115 6.27397 17.2216 5.91601 16.7994 5.59889L18.0006 3.99976C18.5278 4.39576 19.0145 4.84262 19.4535 5.33307C21.0363 7.10159 22 9.43944 22 12C22 14.5605 21.0363 16.8984 19.4535 18.6669C19.0145 19.1573 18.5278 19.6042 18.0006 20.0002L16.7994 18.4011C17.2216 18.084 17.6115 17.726 17.9632 17.3331C19.2306 15.9171 20 14.0496 20 12C20 9.9504 19.2306 8.08291 17.9632 6.6669ZM12 17C14.7614 17 17 14.7614 17 12C17 9.23855 14.7614 6.99998 12 6.99998C9.23858 6.99998 7 9.23855 7 12C7 14.7614 9.23858 17 12 17ZM12 19C15.866 19 19 15.866 19 12C19 8.13398 15.866 4.99998 12 4.99998C8.13401 4.99998 5 8.13398 5 12C5 15.866 8.13401 19 12 19ZM9.9705 10.84H9.0955L8.07 13.5H8.9065L9.0745 12.9505H9.9355L10.107 13.5H10.9995L9.9705 10.84ZM9.505 11.547L9.743 12.3205L9.267 12.3205L9.505 11.547ZM12.3923 10.84H11.3143V13.5H12.4973C13.1717 13.5 13.5088 13.2398 13.5088 12.7195C13.5088 12.5631 13.4703 12.429 13.3933 12.317C13.3163 12.2026 13.2102 12.1245 13.0748 12.0825C13.1705 12.0288 13.2428 11.9541 13.2918 11.8585C13.3432 11.7628 13.3688 11.6531 13.3688 11.5295C13.3688 11.3031 13.286 11.1316 13.1203 11.015C12.957 10.8983 12.7143 10.84 12.3923 10.84ZM12.4308 12.933H12.0668V12.443H12.4308C12.6152 12.443 12.7073 12.5246 12.7073 12.688C12.7073 12.8513 12.6152 12.933 12.4308 12.933ZM12.3328 11.897H12.0668V11.407H12.3328C12.5172 11.407 12.6093 11.4886 12.6093 11.652C12.6093 11.8153 12.5172 11.897 12.3328 11.897ZM14.1753 13.4895C14.3573 13.5338 14.5428 13.556 14.7318 13.556C15.0865 13.556 15.3606 13.4801 15.5543 13.3285C15.7503 13.1745 15.8483 12.9586 15.8483 12.681C15.8483 12.4616 15.7771 12.2843 15.6348 12.149C15.4948 12.0136 15.2673 11.9051 14.9523 11.8235C14.824 11.7908 14.7365 11.7605 14.6898 11.7325C14.6431 11.7021 14.6198 11.6625 14.6198 11.6135C14.6198 11.5575 14.6431 11.5166 14.6898 11.491C14.7365 11.463 14.8088 11.449 14.9068 11.449C15.0258 11.449 15.1518 11.4676 15.2848 11.505C15.4178 11.54 15.5298 11.5866 15.6208 11.645L15.7258 10.9975C15.6255 10.9321 15.4971 10.8808 15.3408 10.8435C15.1845 10.8038 15.0258 10.784 14.8648 10.784C14.5311 10.784 14.2698 10.8598 14.0808 11.0115C13.8941 11.1631 13.8008 11.372 13.8008 11.638C13.8008 11.8596 13.8661 12.0381 13.9968 12.1735C14.1275 12.3088 14.334 12.4126 14.6163 12.485C14.7213 12.513 14.8006 12.5375 14.8543 12.5585C14.908 12.5795 14.9465 12.6028 14.9698 12.6285C14.9955 12.6518 15.0083 12.6821 15.0083 12.7195C15.0083 12.8338 14.9021 12.891 14.6898 12.891C14.5405 12.891 14.3865 12.87 14.2278 12.828C14.0715 12.7836 13.9373 12.7253 13.8253 12.653L13.7553 13.3215C13.8533 13.3891 13.9933 13.4451 14.1753 13.4895Z"
            fill-rule="evenodd"
          />
        </svg>
      </span>
      $100 off
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Pill Snapshots tests should match passed Component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <span
        class="pill"
      >
        $100 off
      </span>
    </div>
  </body>,
  "container": <div>
    <span
      class="pill"
    >
      $100 off
    </span>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Pill Snapshots tests should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="pill"
      >
        $100 off
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="pill"
    >
      $100 off
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
